code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -642,6 +642,97 @@
+	new /obj/item/card/id/advanced/chameleon(src) // 2 TC / Free for nukies
+	var/obj/item/clothing/suit/space/syndicate/spess_suit = pick(GLOB.syndicate_space_suits_to_helmets)
+	new spess_suit(src) // Above allows me to get the helmet from a variable on the object
+	var/obj/item/clothing/head/helmet/space/syndicate/spess_helmet = GLOB.syndicate_space_suits_to_helmets[spess_suit]
+	new spess_helmet(src) // 4 TC for the space gear
+	new /obj/item/tank/jetpack/oxygen/harness(src) // They kinda need this to fly to the cruiser.
+	// Tacticool gear
+	new /obj/item/clothing/shoes/combat(src)
+	new /obj/item/clothing/under/syndicate(src)
+	new /obj/item/clothing/gloves/fingerless(src)
+	new /obj/item/book/manual/nuclear(src) // Very important
+	// The most important part of the kit, the implant that gives them the syndicate faction.
+	new /obj/item/implanter/induction_implant(src)
+	// All in all, 6+3+3+2+5+2+4 = ~25 TC of 'miscellaneous' items.
+	// This is a lot of value for 10 TC, but you have to keep in mind that you NEED someone to get this stuff station-side.
+	// Pretty much all of it is a bad deal for reinforcements or yourself as they already have similar or good-enough alternatives.
+
+/obj/item/implanter/induction_implant
+	name = "implanter (nuclear operative)"
+	desc = "A sterile automatic implant injector. You can see a tiny, somehow legible sticker on the side: 'NOT A BRAINWASH DEVICE'"
+	imp_type = /obj/item/implant/nuclear_operative
+
+/obj/item/implant/nuclear_operative
+	name = "nuclear operative implant"
+	desc = "Registers you as a member of a Syndicate nuclear operative team."
+	implant_color = "r"
+
+/obj/item/implant/nuclear_operative/get_data()
+	var/dat = {"<b>Implant Specifications:</b><BR>
+				<b>Name:</b> Suspicious Implant<BR>
+				<b>Life:</b> UNKNOWN <BR>
+				<b>Implant Details:</b> <BR>
+				<b>Function:</b> Strange implant that seems to resist any attempts at scanning it."}
+	return dat
+
+/obj/item/implant/nuclear_operative/implant(mob/living/target, mob/user, silent = FALSE, force = FALSE)
+	. = ..()
+	if(!. || !isliving(target))
+		return FALSE
+	var/mob/living/living_target = target
+
+	if(IS_NUKE_OP(target)) // this wont proc due to ..() but i guess its good as a just-in-case?
+		if(target == user)
+			to_chat(user, span_userdanger("You're already a nuclear operative, dumbass! The implant disintegrates within you! You feel sick.."))
+			target.Stun(10 SECONDS)
+			target.reagents.add_reagent(/datum/reagent/toxin, 10)
+			return FALSE
+		else
+			to_chat(user, span_notice("You finish implanting [target], but you don't really notice a difference. Huh."))
+			to_chat(target, span_userdanger("Nothing seems to really happen, but you start to feel a little ill.."))
+			target.reagents.add_reagent(/datum/reagent/toxin, 2)
+			return FALSE
+
+	if(!(living_target.mind?.has_antag_datum(/datum/antagonist))) // GTFO. Technically not foolproof but making an ERT member or a paradox clone a nuke op sounds hilarious
```

Comment:
we really should have a list of allowed types in a typecache because of how many things are on the antagonist datum that aren't really malicious antags (like ert members, it's funny but also counter-intuitive to the admin spawning them in)

Reply:
also we should return at the start of the proc if the target has no mind and no client because this is an exercise in futility otherwise

Reply:
i feel like the comment makes a good enough excuse for it, imagine a paradox clone syndie

that said ig youre right whats the solution? a flag for antag datums called CAN_BE_MADE_OPERATIVE?

Reply:
flag might be okay but a trait could work well enough too

Reply:
paradox clone syndie is fine i just don't want an ert becoming a traitor because that directly conflicts with what an admin may be planning through an event, since the only time an ERT spawns is via an admin. it would be really funny but also really weird

Reply:
also stuff like bounty hunters/fugitives? idk it gets murkier around there but it just gets wack. would rather this be an explicit optin via a trait

Reply:
made it a flag